public/js/components/SplitBox.js

```diff
@@ -41,22 +56,34 @@ const SplitBox = React.createClass({
     }
 
     return dom.div(
-      { className: "split-box",
-        style: this.props.style },
+      {
+        className: "split-box",
+        style: this.props.style,
+        ref: "splitBox"
+      },
       dom.div(
-        { className: classnames(
-          { uncontrolled: rightFlex },
-          { controlled: !rightFlex }, { collapsed }),
-          style: { width: rightFlex ? null : width }},
+        {
+          className: classnames(
+            { uncontrolled: rightFlex },
+            { controlled: !rightFlex }, { collapsed }
+          ),
+          style: { width: rightFlex ? null : width },
+          ref: "left"
+        },
         left
       ),
       Draggable({ className: "splitter",
+                  onStop: x => this.onStop(x),
                   onMove: x => this.onMove(x) }),
       dom.div(
-        { className: classnames(
-          { uncontrolled: !rightFlex },
-          { controlled: rightFlex }, { collapsed }),
-          style: { width: rightFlex ? width : null }},
+        {
+          className: classnames(
+            { uncontrolled: !rightFlex },
+            { controlled: rightFlex }, { collapsed }
+          ),
+          style: { width: rightFlex ? width : null },
+          ref: "right"
```

Comment:
It's a lot better to avoid refs if possible. The previous implementation didn't use them; what does this by us? It's a little hard to see from this diff.